analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Constraints/NumberConstraintTest.cs

```diff
@@ -98,15 +99,14 @@ public void ToString_Serialization()
             NumberConstraint.From(0).ToString().Should().Be("Number 0");
             NumberConstraint.From(42).ToString().Should().Be("Number 42");
             NumberConstraint.From(1, 1).ToString().Should().Be("Number 1");
-            NumberConstraint.From(null, null).ToString().Should().Be("Number from * to *");
             NumberConstraint.From(null, 42).ToString().Should().Be("Number from * to 42");
             NumberConstraint.From(42, null).ToString().Should().Be("Number from 42 to *");
             NumberConstraint.From(-1, null).ToString().Should().Be("Number from -1 to *");
             NumberConstraint.From(-4321, -42).ToString().Should().Be("Number from -4321 to -42");
+            NumberConstraint.Empty.ToString().Should().Be("Number from * to *");
```

Comment:
This will need to go back up as `From(null, null)`.

Reply:
I just remove it. `From(null, null)` is covered in `From_Object_OtherTypes()`.